keps/sig-apps/2255-delete-priority/20210112-delete-priority-annotations.yaml

```diff
@@ -0,0 +1,47 @@
+title: KEP Template
+kep-number: 2255
+authors:
+  - "@drbugfinder-work"
+  - "@ahg-g"
+  - "@alculquicondor"
+owning-sig: sig-apps
+participating-sigs:
+status: provisional
+creation-date: yyyy-mm-dd
```

Comment:
nit: update this

Reply:
You mean the date, don't you? I've changed it